app/templates/components/password/forgot-password.hbs

```diff
@@ -1,13 +1,13 @@
 <div class="container">
   <form>
 
-    <p>Enter your email and we'll send you a link to reset your password</p>
+    <p data-test-id="forgot-password-header">Enter your email and we'll send you a link to reset your password</p>
```

Comment:
We've been discussing the usage of https://github.com/simplabs/ember-test-selectors in the past, but I'm nut sure where we're with it now. The addon makes it so these test-only attributes are not present in the final build.

I think, let's not introduce it now, but what you could do is create an issue where we can discuss it and, if we opt into it, create a bunch of other issues. It's a tasks that would be very suitable for newbie volunteer.